frontend/src/components/list/URLList.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { SiteLink } from "src/components/fragments";
+
+interface URLListProps {
+  urls: {
+    url: string;
+    site: {
+      id: string;
+      name: string;
+    } | null;
+  }[];
+}
+
+const URLList: React.FC<URLListProps> = ({ urls }) => (
+  <ul className="URLList">
+    {urls.map((u) => (
+      <li key={u.url}>
+        <SiteLink site={u.site} hideName />
+        <a href={u.url} target="_blank" rel="noreferrer">
```

Comment:
```suggestion
        <a href={u.url} target="_blank" rel="noreferrer noopener">
```
Just in case. It was used before for 'untrusted' links.